crates/full-node/sov-sequencer/src/preferred/mod.rs

```diff
@@ -736,6 +742,7 @@ where
                 "check_readiness",
             )
             .await
+            .map_err(|_| SequencerNotReadyDetails::Shutdown)?
```

Comment:
Do you think it's worth logging error here?

Reply:
Ok, this error cannot be printed. I really hope that if the value is `Unexpected` it is logged somewhere or we will be able to quickly find a root cause.